src/Events/Services/TraceLogService.cs

```diff
@@ -37,11 +42,11 @@ public async Task<string> CreateRegisteredEntry(CloudEvent cloudEvent)
                 };
 
                 await _traceLogRepository.CreateTraceLogEntry(traceLogEntry);
-                return cloudEvent.Id;
+                return cloudEvent.Id ?? string.Empty;
             }
             catch (Exception exception)
             {
-                _logger.LogError(exception, "Error creating trace log entry for registered event: {Message}", exception.Message);
+                _logger.LogError(exception, "Error creating trace log entry for registered event: {Message} {CloudEventId}", exception.Message, cloudEvent.Id);
```

Comment:
Maybe change the order of the two parameters, and something to separate them more clearly in the final message?